src/Models/HydrostaticFreeSurfaceModels/split_explicit_free_surface_kernels.jl

```diff
@@ -14,18 +14,18 @@ using Oceananigans.Operators
 @kernel function split_explicit_free_surface_substep_kernel_1!(grid, Δτ, η, U, V, Gᵁ, Gⱽ, g, Hᶠᶜ, Hᶜᶠ)
     i, j = @index(Global, NTuple)
     # ∂τ(U) = - ∇η + G
-    @inbounds U[i, j, 1] +=  Δτ * (-g * Hᶠᶜ[i, j] * ∂xᶠᶜᶜ(i, j, 1, grid, η) + Gᵁ[i, j, 1])
-    @inbounds V[i, j, 1] +=  Δτ * (-g * Hᶜᶠ[i, j] * ∂yᶜᶠᶜ(i, j, 1, grid, η) + Gⱽ[i, j, 1])
+    @inbounds U[i, j, 1] +=  Δτ * (-g * Hᶠᶜ[i, j] * ∂xᶠᶜᶠ(i, j, grid.Nz+1, grid, η) + Gᵁ[i, j, 1])
+    @inbounds V[i, j, 1] +=  Δτ * (-g * Hᶜᶠ[i, j] * ∂yᶜᶠᶠ(i, j, grid.Nz+1, grid, η) + Gⱽ[i, j, 1])
 end
 
 @kernel function split_explicit_free_surface_substep_kernel_2!(grid, Δτ, η, U, V, η̅, U̅, V̅, velocity_weight, free_surface_weight)
     i, j = @index(Global, NTuple)
     # ∂τ(η) = - ∇⋅U
-    @inbounds η[i, j, 1] -=  Δτ * div_xyᶜᶜᶜ(i, j, 1, grid, U, V)
+    @inbounds η[i, j, grid.Nz+1] -=  Δτ * div_xyᶜᶜᶜ(i, j, 1, grid, U, V)
```

Comment:
```suggestion
    @inbounds η[i, j, grid.Nz+1] -=  Δτ * div_xyᶜᶜᶜ(i, j, grid.Nz, grid, U, V)
```

??

Reply:
correct! (technically I am not sure how to treat `ReducedFields` probably we should still go at `grid.Nz+1`?)

Reply:
but doesn't `div_xyᶜᶜᶜ` involve `Azᶜᶜᶜ(i, j, k)`? That should be at `k = Nz`, right?

Reply:
you're right. then we should write a `div_xyᶜᶜᶠ` for fields at `Face`s

Reply:
Should we have had a test to pick this up?

Reply:
I don't think this is correct. `U` and `V` are _reduced_ fields with only one grid point. This differs from _sliced_ fields, which exist at a particular model level. `k=1` is better.

Reply:
The problem with `k=Nz` is that it suggests incorrectly that `U` and `V` depend on z.

Reply:
@navidcy didn't realize that `U` and `V` were barotropic; that's why he made his comment (that's why we need to use `k=1` and also include a comment.)